packages/bpk-component-calendar/readme.md

```diff
@@ -283,7 +283,7 @@ If set to false, the calendar is of fluid width and will take up the space of it
 
 #### formatDateFull
 
-A function to format a full, human-readable date, for example: "Friday, 13th January 2017":
+A function to format a full, human-readable date, for example: "Friday, 13th January 2018":
```

Comment:
I've updated this one as well, backpack is a well maintained package!

Reply:
Ah, but that's not a valid date as the 13th is next Saturday 😉

Reply:
I tried to be clever and failed miserably!